kinematic_chain_base_link: base_link
contact_point_offset: 0.05
action_name: folding_controller
initial_wait_time: 2
base_align: -y
wiggle_max_time: 15
degree_n: 0.1
degree_r: 1.0

kdl_manager:
  eps: 0.001
  max_tf_attempts: 10

rod_arm:
  kdl_eef_frame: "left_link_7"
  gripping_frame: "suction_cup_tip" #"gripper_l_finger_l"
  has_ft_sensor: false
  sensor_frame: ""
  sensor_topic: ""
  align_axis: z

surface_arm:
  kdl_eef_frame: "right_link_7"
  gripping_frame: "r_gripping_point"
  has_ft_sensor: true
  sensor_frame: "sensor_link"
  sensor_topic: "/ft_sensor/ft_compensated"
  align_axis: y

kf_estimator:
  Q:
    data: [1.01, 0, 0, 0, 1.01, 0, 0, 0, 1.01]
  R:
    data: [0.01, 0, 0, 0, 0.01, 0, 0, 0, 0.01]

relative_pose_controller:
  vd_max: 0.01
  wd_max: 0.04
  position_gain: -0.3
  orientation_gain: 0.5 # 0.4

absolute_pose_controller:
  vd_max: 0.0
  wd_max: 0.025
  position_gain: -0.1
  orientation_gain: -0.1

ects_controller:
  alpha: 0.5
  beta: 1
  inverse_damping: 0.001

grasp:
  translational_axis: -y
  rotational_axis: -x

adaptive_estimator:
  alpha_force: 0.005 # 0.01
  beta_force: 0.01
  force_slack: 0.0
  alpha_adapt_t: 0.0
  alpha_torque_t: 0.15 # 0.3
  alpha_torque_n: 0.5
  beta_torque_t: 0.4 # 0.4
  beta_torque_n: 1.5
  wiggle_amplitude: 0 #-0.08
  wiggle_frequency: 0.1
  alpha_adapt_r: 0
  torque_slack: 0.0
  max_force: 3
  max_torque: 0.04
